package.json

```diff
@@ -9,14 +9,12 @@
     "prettier": "prettier --write src/**/*.js",
     "deploy": "npm run build && now deploy --static ./public/"
   },
-  "pre-commit": [
-    "prettier"
-  ],
   "dependencies": {
     "aframe": "0.7.0",
-    "aframe-input-mapping-component": "https://github.com/fernandojsg/aframe-input-mapping-component",
+    "aframe-input-mapping-component": "https://github.com/fernandojsg/aframe-input-mapping-component#6ebc38f",
     "aframe-teleport-controls": "https://github.com/netpro2k/aframe-teleport-controls#feature/teleport-origin",
-    "naf-janus-adapter": "^0.1.4",
+    "minijanus": "^0.1.6",
+    "naf-janus-adapter": "https://github.com/mozilla/naf-janus-adapter#new-janus-api",
```

Comment:
This should point to a new npm package version when we publish it (mozilla/naf-janus-adapter#6)

Reply:
Updated